plugins/source/aws/docs/index.md

```diff
@@ -71,7 +71,7 @@ export AWS_PROFILE=myprofile
 
 or, configure your desired profile in the `local_profile` field of your CloudQuery `cloudquery.yml`:
 
-```yml title="cloudquery.yml"
```

Comment:
Renamed to `yaml` as `yml` is not supported by Nextra: https://github.com/FormidableLabs/prism-react-renderer/blob/4d8019490a3ad89c310d877a9ae3130084261bc9/src/vendor/prism/includeLangs.js